README.md

```diff
@@ -12,6 +12,9 @@ If you need any help with the bot or you think you have found a bug please join
 If you want to help FlareBot's hosting and development please use [this link](https://www.paypal.me/FlareBot). Thank you!
 
 ## Credits
-* Initially started by [WalshyDev(@bwfcwalshy#1284)](https://github.com/WalshyDev/) and [ArsenArsen(@Arsen#7525)](https://github.com/ArsenArsen/)
+* Initially started by [WalshyDev(@Storm#0681)](https://github.com/WalshyDev/) and [ArsenArsen(@Arsen#7525)](https://github.com/ArsenArsen/)
```

Comment:
As said by @WalshyDev this should not be updated until he returns into bwfcwalshy. Don't bother